webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const EmptyState = (props) => {
+  const {
+    icon = 'add-circle-o',
+    header,
+    description,
+    customDocumentation,
+    documentationLabel = 'For more information please see',
+    documentationButton = 'Documentation',
+    docUrl,
+    action,
+    secondayActions,
+  } = props;
+  const defaultDocumantion = __(`${documentationLabel} <a href=${docUrl}>${documentationButton}</a>`);
```

Comment:
This is not going to work correctly. The string gets interpreted first so you'll get a string `"For more information..."` but translation files will contain key `"${documentationLabel} <a href=${docUrl}>${documentationButton}</a>"` and that's not going to match.

We need to use `sprintf` (for instance like here https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/reduxFormI18n.js#L12)

Reply:
Nitpick, spelling is incorrect, should be documentation not documantion.